SerialPrograms/Source/ZeldaTotK/Programs/ZeldaTotK_BowItemDuper.cpp

```diff
@@ -0,0 +1,167 @@
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ */
+
+#include "Common/Cpp/PrettyPrint.h"
+#include "CommonFramework/Exceptions/OperationFailedException.h"
+#include "CommonFramework/Notifications/ProgramNotifications.h"
+#include "CommonFramework/InferenceInfra/InferenceRoutines.h"
+#include "CommonFramework/Tools/StatsTracking.h"
+#include "CommonFramework/Tools/VideoResolutionCheck.h"
+#include "NintendoSwitch/NintendoSwitch_Settings.h"
+#include "NintendoSwitch/Commands/NintendoSwitch_Commands_PushButtons.h"
+#include "NintendoSwitch/Commands/NintendoSwitch_Commands_ScalarButtons.h"
+#include "NintendoSwitch/Programs/NintendoSwitch_GameEntry.h"
+
+
+#include "ZeldaTotK/ZeldaTotK_Settings.h"
+
+#include "ZeldaTotK_BowItemDuper.h"
+
+namespace PokemonAutomation {
+namespace NintendoSwitch {
+namespace ZeldaTotK {
+
+// using namespace Pokemon;
+
+BowItemDuper_Descriptor::BowItemDuper_Descriptor()
+    : SingleSwitchProgramDescriptor(
+        "ZeldaTotK:BowItemDuper",
+        "Zelda: TotK", "Bow Item Duper",
+        "ComputerControl/blob/master/Wiki/Programs/ZeldaTotK/BowItemDuper.md",
+        "Use the Bow Swap Glitch to farm any fusable items.",
+        FeedbackType::NONE,
+        AllowCommandsWhenRunning::DISABLE_COMMANDS,
+        PABotBaseLevel::PABOTBASE_12KB
+    )
+{}
+
+struct BowItemDuper_Descriptor::Stats : public StatsTracker {
+    Stats()
+        : dupe_attempts(m_stats["Dupe Attempts"])
+        , errors(m_stats["Errors"])
+    {
+        m_display_order.emplace_back("Dupe Attempts");
+        m_display_order.emplace_back("Errors", true);
+    }
+    std::atomic<uint64_t>& dupe_attempts;
+    std::atomic<uint64_t>& errors;
+};
+std::unique_ptr<StatsTracker> BowItemDuper_Descriptor::make_stats() const {
+    return std::unique_ptr<StatsTracker>(new Stats());
+}
+
+BowItemDuper::BowItemDuper()
+    : ATTEMPTS(
+        "<b>Duplication Attempts:</b><br>The number of times you wish to run this routine.",
+        LockWhileRunning::UNLOCKED,
+        100
+    )
+    , TICK_DELAY(
+        "<b>Menu Delay:</b><br>Adjustable delay for exiting and reentering the menu. This should not typically be lower than 5 nor exceed 25 for successful results. Too low and the game will eat inputs and put the program in an irrecoverable state. Too high and the dupes will not be successful.",
+        LockWhileRunning::UNLOCKED,
+        15
+    )
+    , GO_HOME_WHEN_DONE(false)
+    , NOTIFICATION_STATUS_UPDATE("Status Update", true, false, std::chrono::seconds(3600))
+    , NOTIFICATIONS({
+        &NOTIFICATION_STATUS_UPDATE,
+        &NOTIFICATION_PROGRAM_FINISH,
+        // &NOTIFICATION_ERROR_FATAL,
+        })
+{
+    PA_ADD_OPTION(ATTEMPTS);
+    PA_ADD_OPTION(TICK_DELAY);
+    PA_ADD_OPTION(GO_HOME_WHEN_DONE);
+    PA_ADD_OPTION(NOTIFICATIONS);
+}
+
+void BowItemDuper::program(SingleSwitchProgramEnvironment& env, BotBaseContext& context) {
+    assert_16_9_720p_min(env.logger(), env.console);

```

Comment:
Can comment this out as well since we're not using video.

Reply:
Commented out in latest commit